test/e2e/deployment.go

```diff
@@ -110,24 +111,26 @@ func newDeployment(deploymentName string, replicas int, podLabels map[string]str
 func testNewDeployment(f *Framework) {
 	ns := f.Namespace.Name
 	c := f.Client
```

Comment:
VerifyPods and other functions still use the unversioned client. I add TODO to remove it.